doc/REST-interface.md

```diff
@@ -89,11 +89,13 @@ Only supports JSON as output format.
 * softforks : (array) status of softforks in progress
 
 #### Query UTXO set
-`GET /rest/getutxos/<checkmempool>/<txid>-<n>/<txid>-<n>/.../<txid>-<n>.<bin|hex|json>`
+`GET /rest/getutxos/<TXID>-<N>/<TXID>-<N>/.../<TXID>-<N>.<bin|hex|json>` <br>
+`GET /rest/getutxos/checkmempool/<TXID>-<N>/<TXID>-<N>/.../<TXID>-<N>.<bin|hex|json>`
 
-The getutxo command allows querying of the UTXO set given a set of outpoints.
-See BIP64 for input and output serialisation:
-https://github.com/bitcoin/bips/blob/master/bip-0064.mediawiki
+The getutxos endpoint allows querying the UTXO set given a set of outpoints.
```

Comment:
if you retouch
```suggestion
The getutxos endpoint allows querying the UTXO set, given a set of outpoints.
```
and also `s/option/option,/

Reply:
Thanks, done.